darts/utils/data/tabularization.py

```diff
@@ -775,10 +784,31 @@ def create_lagged_component_names(
 
         components = get_single_series(variate).components.tolist()
         if isinstance(variate_lags, dict):
+            if "default_lags" in variate_lags:
```

Comment:
As the `RegressionModel` already handles it, I wanted to avoid code duplication (and circular imports) while preventing unexpected behavior if a user calls this method with parameters not directly extracted from a model.